onnx/__init__.py

```diff
@@ -91,17 +98,24 @@ def _get_file_path(f: Union[IO[bytes], str]) -> Optional[str]:
     return None
 
 
-def _serialize(proto: Union[bytes, google.protobuf.message.Message]) -> bytes:
+def _serialize(
+    proto: Union[bytes, google.protobuf.message.Message], format_: _SupportedFormat
+) -> bytes:
     """Serialize a in-memory proto to bytes.
 
     Args:
         proto: An in-memory proto, such as a ModelProto, TensorProto, etc
+        format: The serialization format. Can be one of :py:const:`_SupportedFormat`.
 
     Returns:
         Serialized proto in bytes.
     """
     if isinstance(proto, bytes):
         return proto
+    if format_ == "textformat":
+        return google.protobuf.text_format.MessageToString(proto).encode("utf-8")
```

Comment:
I wonder if the encoding should be exposed as a parameter of the function.

Reply:
I am inclined to suggest we only support one single encoding. I don’t think there is a point of supporting others?

Reply:
Done. PATL